src/axom/primal/operators/winding_number.hpp

```diff
@@ -454,26 +454,9 @@ int winding_number(const Point<T, 3>& query,
 }
 
 #ifdef AXOM_USE_MFEM
-/*!
```

Comment:
why remove the method docs here?

Reply:
"we hates it! we HATES the documentation! it burns (gollum, gollum)"

Reply:
or, as a Klingon would say: "Documentation is for the weak"

Reply:
I've got no clue where that comment went, especially since the method was from another PR entirely. I'm glad it's back though, it was pretty precious to me.

Reply:
Thanks!